azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -1261,6 +1205,13 @@ public void close() {
     }
     scheduleManager.shutdown();
     executorManager.shutdown();
+    try {
+      server.stop();
+    } catch (Throwable t) {
```

Comment:
@HappyRay 
The method call itself throws `Exception` so we cannot just catch `RuntimeException`. I have already made the suggested changes and pushed it #1019